grails-shell/build.gradle

```diff
@@ -62,18 +62,17 @@ dependencies {
         exclude group: "org.slf4j", module:"jcl-over-slf4j"
         exclude group: "commons-codec", module:"commons-codec"
     }
-    implementation "commons-codec:commons-codec:$commonsCodecVersion"
+    implementation "commons-codec:commons-codec"
```

Comment:
`commmons-codec:commons-codec` is not used in `grails-shell`

Reply:
removed